### PR TITLE
add missed letters ( ၌ ၍ ၎ ၏ )  and symbols ( ၊ ။ ) - 0x104a to 0x104f -

### PR DESCRIPTION
**Pull request by pndaza**

### CODE REVIEW

File: training/validate_myanmar.cpp [hunk 1]
```diff
@@ -135,6 +135,7 @@ bool ValidateMyanmar::ConsumeOptionalSignsIfPresent() {
 /* static */
 bool ValidateMyanmar::IsMyanmarLetter(char32 ch) {
   return (0x1000 <= ch && ch <= 0x102a) || ch == 0x103f ||
+         (0x104c <= ch && ch <= 0x104f) ||
```
Comment (stweil) [inline training/validate_myanmar.cpp]: Is it better to use a separate condition like you have done it, or could it be combined with the next line? The new condition ends at 0x104f and the next condition starts at 0x1050.

**Comment by stweil** [comment]
@pndaza, could you please update your clone of tesseract and fix the conflicts for this pull request?

**Comment by amitdo** [comment]
https://unicode.org/charts/PDF/U1000.pdf

**Comment by Shreeshrii** [comment]
@stweil Can this be merged?

**Comment by stweil** [comment]
I have no open questions, but don't know enough of that language to decide whether it is needed and can be merged. Can you or @amitdo review and approve it?

**Pull request merged**